lib/timber/events/http.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Timber.Events.HTTP do
+  @moduledoc """
+  Utility module for common HTTP related events.
+  """
+
+  def normalize_method(method) do
```

Comment:
Is there a reason this wouldn't go in `Timber.Utils`? Seems inappropriate for the `Timber.Events` namespace